frontend/controller/serviceworkers/sw-primary.js

```diff
@@ -114,10 +114,22 @@ const setupRootState = () => {
 
 sbp('okTurtles.events/on', CHELONIA_RESET, setupRootState)
 
+const broadcastMessage = (...args) => {
+  self.clients.matchAll()
+    .then((clientList) => {
+      // eslint-disable-next-line require-await
+      return Promise.all(clientList.map(async (client) => {
+        return client.postMessage(...args)
```

Comment:
Why is this an `async` function without an `await`? What's the reason for marking it `async` like this?

(Can you add a comment above line 120 explaining?)

Reply:
Not really sure why that lint rule is there to begin with, as there are many reasons to do so. In this case, it's for `postMessage` to be run even if there's an error.

See `[1,2,3].map(async (v) => {throw v})` (1, 2, 3 are thrown) vs `[1,2,3].map(async (v) => {throw v})` (1 is thrown, then it stops).

Reply:
> See `[1,2,3].map(async (v) => {throw v})` (1, 2, 3 are thrown) vs `[1,2,3].map(async (v) => {throw v})` (1 is thrown, then it stops).

Did you mean something else for the second example? Those two code blocks are the same.